monix-reactive/shared/src/main/scala/monix/reactive/Observable.scala

```diff
@@ -1145,6 +1146,35 @@ abstract class Observable[+A] extends Serializable { self =>
   final def doOnStart(cb: A => Unit): Observable[A] =
     self.liftByOperator(new DoOnStartOperator[A](cb))
 
+  /** Executes the given callback only for the first element generated
+    * by the source Observable, useful for doing a piece of
+    * computation only when the stream starts.
+    *
+    * @return a new Observable that executes the specified callback
+    *         only for the first element
+    * @see [[doOnStart]] for a simpler version that doesn't allow
+    *     asynchronous execution, or
+    *     [[doOnStartEval]] for a version that can do ev
```

Comment:
Oh, and sorry for the nitpicks but something went wrong with the formatting here (looks better on `doOnStartEval` version)

Reply:
yeah, totally, my bad!